app/controllers/repos_controller.rb

```diff
@@ -26,10 +26,16 @@ def show
 
       @charts = @benchmark_type.benchmark_result_types.map do |benchmark_result_type|
         cache_key = "#{BenchmarkRun.charts_cache_key(@benchmark_type, benchmark_result_type)}:#{@benchmark_run_display_count}"
+        ruby_version_cache_key = "ruby_version_#{cache_key}"
 
-        if (columns = $redis.get(cache_key))
+        if (columns = $redis.get(cache_key)) && (ruby_versions = $redis.get(ruby_version_cache_key))
```

Comment:
That makes sense. I changed it to use a generic `@versions`, so it should work with other repos as well now.